hs_core/templates/pages/create-resource.html

```diff
@@ -48,14 +48,14 @@
         <div class="col-sm-10">
             <select class="form-control" name="resource-type" id="resource-type">
                 <option value="GenericResource">Generic</option>
-                <option value="RasterResource">Geographic Raster</option>
-                <option value="RefTimeSeries">HIS Referenced Time Series</option>
-                <option value="ModelProgramResource">Model Program</option>
-                <option value="ModelInstanceResource">Model Instance</option>
-                <option value="SWATModelInstanceResource">SWAT Model Instance</option>
-                <option value="NetcdfResource">Multidimensional (NetCDF)</option>
-                <option value="TimeSeriesResource">Time Series</option>
-                <option value="ToolResource">Application</option>
+{#                <option value="RasterResource">Geographic Raster</option>#}
+{#                <option value="RefTimeSeries">HIS Referenced Time Series</option>#}
+{#                <option value="ModelProgramResource">Model Program</option>#}
+{#                <option value="ModelInstanceResource">Model Instance</option>#}
+{#                <option value="SWATModelInstanceResource">SWAT Model Instance</option>#}
+{#                <option value="NetcdfResource">Multidimensional (NetCDF)</option>#}
+{#                <option value="TimeSeriesResource">Time Series</option>#}
+{#                <option value="ToolResource">Application</option>#}

```

Comment:
I made these hidden now in this branch since the new access control only works for 'GenericResource'. Once we merge the resource refactoring code these can again be turned and tested to see access control works for other resource types.